bin/check-aggregate.rb

```diff
@@ -130,6 +130,16 @@ class CheckAggregate < Sensu::Plugin::Check::CLI
          boolean: true,
          default: false
 
+  option :stale,
```

Comment:
if we are going to use `--stale-count` then I think this should be `--stale-percentage` as looking at the 2 options before reading the description I assumed it was any stale seeing both `--stale` and `--stale-count`